libraries/AP_GPS/AP_GPS_MAV.cpp

```diff
@@ -71,6 +71,8 @@ void AP_GPS_MAV::handle_msg(const mavlink_message_t &msg)
             loc.lng = packet.lon;
             if (have_alt) {
                 loc.alt = packet.alt * 100; // convert to centimeters
+            } else {
+                state.have_altitude = false;
```

Comment:
This could change over time, so we might need to set just `state.have_altitude = have_alt;`?

There might be a reason we shouldn't do that, in which case a comment would be good :-)

Reply:
you are right, there is no need to set it here only to false.
changed that